secrets/secrets.go

```diff
@@ -116,6 +116,15 @@ func isSecretIgnored(secret *reporting.Secret, ignoredIds *[]string) bool {
 	return false
 }
 
+func isFileNameExcluded(secret *reporting.Secret, excludedFileNames *[]string) bool {
+	for _, excludedFileName := range *excludedFileNames {
+		if filepath.Base(secret.Source) == excludedFileName || filepath.Dir(secret.Source) == excludedFileName {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
Think about that- when do we handling the files? And as you asked on #198- are the "Total Items Scanned" contains the excluded items?

I'm trying to say that first, I think you should move the exclude file mechanism to the plugin. The plugin must not send the file to search secrets if it is ignored.

Second, I'm not sure if we need to implement this only for filesystem or for all the plugins, I'm waiting for answer [here](https://github.com/Checkmarx/2ms/issues/194#issuecomment-1892675187).

Reply:
@baruchiru
Hey, first of all - thanks for the response, I will work on the comments soon.
I think that we should implement it also in every other platform the tool supports, also if it wasn't the issue creator's intention - it will be useful anyway and it can't hurt (in my opinion). With that being said, I'm not sure I will manage to do it by myself, also because I'm lacking access to confluence (for example) in order to debug that, so maybe we should open a separate issue for each task...